pkg/operator/e2e/main_test.go

```diff
@@ -347,29 +355,6 @@ alerting:
                 own_namespace: false
                 names:
                     - {namespace}
```

Comment:
I couldn't comment here but I want to know if the `kubernetes_sd_configs` here is constructed by default - or if it's  something added by the users additionally.

Reply:
It's in the test because we configure an [extra alertmanager](https://github.com/GoogleCloudPlatform/prometheus-engine/blob/7106c5142e22edb3ff337a98fdfb525d1609e258/pkg/operator/e2e/main_test.go#L209) in the test harness.

I'm leaving non-managed PAMs on k8s SD. I suppose we could change that in the future as well, but probably not necessary right now.

Reply:
Agreed - I also read up [1] and [2] and understand now it's for configuring additional alertmanager endpoints. Agree that we can leave them out of this PR and decide later as it can be a separate decision.

Thanks for the clarification and working it through with me

[1] https://github.com/GoogleCloudPlatform/prometheus-engine/blob/main/doc/api.md#alertmanagerendpoints
[2] https://cloud.google.com/stackdriver/docs/managed-prometheus/rules-managed#self-deployed_alertmanager